<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonalDataWidget</class>
 <widget class="QWidget" name="PersonalDataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2423</width>
    <height>777</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_13">
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFrame" name="frame">
         <property name="styleSheet">
          <string notr="true">border:0.8px solid rgba(77,169,255);
border-radius:8px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item row="0" column="0">
           <widget class="QFrame" name="frame_2">
            <property name="styleSheet">
             <string notr="true">border:0.6px solid rgba(77,169,255,0.8);
border-radius:6px;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <property name="leftMargin">
              <number>1</number>
             </property>
             <property name="topMargin">
              <number>1</number>
             </property>
             <property name="rightMargin">
              <number>1</number>
             </property>
             <property name="bottomMargin">
              <number>1</number>
             </property>
             <item row="0" column="0">
              <widget class="QWidget" name="widget_left" native="true">
               <property name="minimumSize">
                <size>
                 <width>350</width>
                 <height>400</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">*{
background: transparent;
border:1px solid rgba(77,169,255,0.6);
border-radius:5px;
}</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="styleSheet">
          <string notr="true">border:0.8px solid rgba(77,169,255);
border-radius:8px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item row="0" column="0">
           <widget class="QFrame" name="frame_4">
            <property name="styleSheet">
             <string notr="true">border:0.6px solid rgba(77,169,255,0.8);
border-radius:6px;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <property name="leftMargin">
              <number>1</number>
             </property>
             <property name="topMargin">
              <number>1</number>
             </property>
             <property name="rightMargin">
              <number>1</number>
             </property>
             <property name="bottomMargin">
              <number>1</number>
             </property>
             <item row="0" column="0">
              <widget class="QWidget" name="widget_right" native="true">
               <property name="minimumSize">
                <size>
                 <width>350</width>
                 <height>400</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">*{
background: transparent;
border:1px solid rgba(77,169,255,0.6);
border-radius:5px;
}</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QFrame" name="frame_5">
       <property name="styleSheet">
        <string notr="true">border:0.8px solid rgba(77,169,255);
border-radius:8px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item row="0" column="0">
         <widget class="QFrame" name="frame_6">
          <property name="styleSheet">
           <string notr="true">border:0.6px solid rgba(77,169,255,0.8);
border-radius:6px;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <property name="leftMargin">
            <number>1</number>
           </property>
           <property name="topMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>1</number>
           </property>
           <property name="bottomMargin">
            <number>1</number>
           </property>
           <item row="0" column="0">
            <widget class="QWidget" name="widget_line_left" native="true">
             <property name="styleSheet">
              <string notr="true">*{
background: transparent;
border:1px solid rgba(77,169,255,0.6);
border-radius:5px;
}
</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_7">
       <property name="styleSheet">
        <string notr="true">border:0.8px solid rgba(77,169,255);
border-radius:8px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item row="0" column="0">
         <widget class="QFrame" name="frame_8">
          <property name="styleSheet">
           <string notr="true">border:0.6px solid rgba(77,169,255,0.8);
border-radius:6px;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <property name="leftMargin">
            <number>1</number>
           </property>
           <property name="topMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>1</number>
           </property>
           <property name="bottomMargin">
            <number>1</number>
           </property>
           <item row="0" column="0">
            <widget class="QWidget" name="widget_bar_left" native="true">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">*{
background: transparent;
border:1px solid rgba(77,169,255,0.6);
border-radius:5px;
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QFrame" name="frame_9">
       <property name="styleSheet">
        <string notr="true">border:0.8px solid rgba(77,169,255);
border-radius:8px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_9">
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item row="0" column="0">
         <widget class="QFrame" name="frame_10">
          <property name="styleSheet">
           <string notr="true">border:0.6px solid rgba(77,169,255,0.8);
border-radius:6px;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_10">
           <property name="leftMargin">
            <number>1</number>
           </property>
           <property name="topMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>1</number>
           </property>
           <property name="bottomMargin">
            <number>1</number>
           </property>
           <item row="0" column="0">
            <widget class="QWidget" name="widget_line_right" native="true">
             <property name="styleSheet">
              <string notr="true">*{
background: transparent;
border:1px solid rgba(77,169,255,0.6);
border-radius:5px;
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_11">
       <property name="styleSheet">
        <string notr="true">border:0.8px solid rgba(77,169,255);
border-radius:8px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_11">
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item row="0" column="0">
         <widget class="QFrame" name="frame_12">
          <property name="styleSheet">
           <string notr="true">border:0.6px solid rgba(77,169,255,0.8);
border-radius:6px;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_12">
           <property name="leftMargin">
            <number>1</number>
           </property>
           <property name="topMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>1</number>
           </property>
           <property name="bottomMargin">
            <number>1</number>
           </property>
           <item row="0" column="0">
            <widget class="QWidget" name="widget_bar_right" native="true">
             <property name="styleSheet">
              <string notr="true">*{
background: transparent;
border:1px solid rgba(77,169,255,0.6);
border-radius:5px;
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
